packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -265,6 +265,12 @@ export const ValuedReferenceAutocomplete = ({
           placeholder={placeholder}
           error={widget.diagnostics.length > 0}
           helperText={widget.diagnostics[0]?.message}
+          inputProps={{
```

Comment:
It seems strange to add a new `inputProps` while there is already an `InputProps` property declared on the component.
Did you try to add your new style properties in the existing `InputProps`?

Reply:
Yes I tried but the "style" is applied to the main "div" of the component. In my case I wanted to reduce the space of the input element.

Current implementation
![image](https://github.com/eclipse-sirius/sirius-web/assets/4439675/a13cac1d-9a19-4b85-a89e-d42d4c19ef3e)

With style class modification
![image](https://github.com/eclipse-sirius/sirius-web/assets/4439675/e09b1530-c41c-4c58-85f0-221a66694004)

Reply:
Same reason than above.
The _InputProps_ are the props of the _div_ owning the _input_ whereas _inputProps_ are the props used for the _input_ element itself.

Reply:
Ok